test/functional/rpc_fundrawtransaction.py

```diff
@@ -57,6 +67,12 @@ def run_test(self):
         watchonly_amount = Decimal(200)
         self.nodes[3].importpubkey(watchonly_pubkey, "", True)
         watchonly_txid = self.nodes[0].sendtoaddress(watchonly_address, watchonly_amount)
+
+        # Lock UTXO so nodes[0] doesn't accidentally spend it
+        watchonly_vout = find_vout_for_address(self.nodes[0], watchonly_txid, watchonly_address)
+        assert watchonly_vout != -1 # it must exist
```

Comment:
You can move the assert into `find_vout_for_address`.

If you don't want to do that, note that python has no types, so you don't have to return an integer. `None` works just fine.

Reply:
You mean `assert watchonly_vout is not None` and not return anything in `find_v..`?

Reply:
Actually nevermind, I'll move the assert instead.